seekinternational/seek-asia-style-guide
# fix(JobCard): Fix JobCard margin bottom, add isSplitView

1. Remove JobCard margin bottom
2. Add isSplitView. For handling split view style and content.

BREAKING CHANGE:
```
<JobCard
   {...someProps}
   isSplitView
/>
```

How to test:
In Job Card, checked Variation. 
1. Get metadata in right hand side, shelf link in right, have selling points
![image](https://user-images.githubusercontent.com/45160649/51371292-bdea2980-1b34-11e9-8b3d-c19ce6d09000.png)
2. Check split view with variation, no metadata, shelf link in left, no selling points
![image](https://user-images.githubusercontent.com/45160649/51371400-073a7900-1b35-11e9-884d-e723f85e183b.png)
3. Check JobCard margin-bottom, it should be 0
